src/slack.coffee

```diff
@@ -242,4 +245,29 @@ class SlackBot extends Adapter
     channel = @client.getChannelGroupOrDMByName envelope.room
     channel.setTopic strings.join "\n"
 
+  customMessage: (data) =>
+    channel = @client.getChannelGroupOrDMByName data.message.envelope.room
+
+    msg = []
+    data.attachments = data.content if data.content and not data.attachments
+    data.attachments = [data.attachments] unless Array.isArray data.attachments
```

Comment:
If you want scripts to be compatible with @inkel's hubot-slack-attachment script, you will want to use `data.channel || data.message.room`. [hubot-bitbucket-pr][hbp] for example will not work with your current patch (@paulhammond I have sent you a patch).

  [hbp]: https://github.com/hubot-scripts/hubot-bitbucket-pr